cli/printer.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const Printer = {
```

Comment:
We could leave that for the next round, when we blend in the extension's HTML reporter.